cmd/av/helpers.go

```diff
@@ -18,18 +18,21 @@ var cachedRepo *git.Repo
 
 func getRepo() (*git.Repo, error) {
 	if cachedRepo == nil {
-		cmd := exec.Command("git", "rev-parse", "--show-toplevel")
+		cmd := exec.Command("git", "rev-parse", "--path-format=absolute", "--git-dir")
+
 		if rootFlags.Directory != "" {
 			cmd.Dir = rootFlags.Directory
 		}
-		toplevel, err := cmd.Output()
+		gitPath, err := cmd.Output()
 		if err != nil {
 			return nil, errors.Wrap(
 				err,
-				"failed to determine repo toplevel (are you running inside a Git repo?)",
+				"failed to determine current worktree's git directory (are you running inside a Git repo?)",
```

Comment:
Most users won't know what a worktree is, let's try to make this error a little bit more friendly.
```suggestion
				"failed to find .git directory (are you running inside a Git repo?)",
```